src/GeolocatorService.Abstractions/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("GeolocatorService")]
```

Comment:
What do you need those for?

Reply:
I do not, I was looking at DynamicMVVM's Abstractions package and basically started by copying stuff over, but looking more closely at what this does, there are no internal methods in the project so this is useless >> will remove